Report.md

```diff
@@ -16,6 +20,8 @@ Problem Definition
 * The Call Management Centre currently provides a suboptimal user experience, due to wait times and poor customer-RM matching, leading to user disatisfaction
```

Comment:
Are we going to keep all of these dot points if it's been summarized above?

Reply:
These were just dot points from planning, I am not sure if we still need them?

Reply:
I reckon just remove them